forge-gui/res/cardsfolder/upcoming/thijarian_witness.txt

```diff
@@ -0,0 +1,10 @@
+Name:Thijarian Witness
+ManaCost:1 G
+Types:Creature Alien Cleric
+PT:0/4
+K:Flash
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.Other+attacking,Creature.Other+blocking | IsPresent$ Card.Other+attacking,Card.Other+blocking | PresentCompare$ EQ1 | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever another creature dies, if it was attacking or blocking alone, exile it and investigate. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")

```

Comment:
Couldn't figure out how to properly enforce the "if it was attacking or blocking alone" part.